code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -722,9 +722,12 @@
 	if(wires.is_cut(WIRE_AVOIDANCE)) // usually just bumps, but if the avoidance wire is cut, knocks them over.
 		if(iscyborg(L))
 			visible_message("<span class='danger'>[src] bumps into [L]!</span>")
-		else if(L.Knockdown(8 SECONDS))
-			log_combat(src, L, "knocked down")
-			visible_message("<span class='danger'>[src] knocks over [L]!</span>")
+		else
+			L.Knockdown(4 SECONDS)
```

Comment:
```suggestion
		else if(L.Knockdown(4 SECONDS))
```
Keep this check, as some living things can absorb the stun, and then the message would be erroneous.